palace/linalg/mumps.cpp

```diff
@@ -34,7 +36,8 @@ MumpsSolver::MumpsSolver(MPI_Comm comm, mfem::MUMPSSolver::MatType sym,
   }
   else
   {
-    SetReorderingStrategy(mfem::MUMPSSolver::AUTOMATIC);  // MUMPS should have good defaults
+    // SetReorderingStrategy(mfem::MUMPSSolver::AUTOMATIC);  // Should have good default
```

Comment:
Going to keep this since the only reason for not using `AUTOMATIC` is that recent MUMPS seems to have an issue (segfault) with METIS (the AUTOMATIC selection) occasionally but works fine with PORD. But in principle METIS should be the preferred choice.